whiteboard/handlers/ExecuteViewSubmitHandler.ts

```diff
@@ -93,7 +96,45 @@ export class ExecuteViewSubmitHandler {
                                 .getMessageReader()
                                 .getRoom(messageId);
 
-                                if(room){
+                            var checkName = 0;
+                            if (room) {
+                                const checkBoardName =
+                                    await checkBoardNameByRoomId(
+                                        this.read.getPersistenceReader(),
+                                        room.id,
+                                        newBoardname
+                                    );
+
+                                // Board name exists already
+                                if (checkBoardName == 1) {
+                                    checkName = 1;
+                                }
+                            }
+
+                            if (checkName == 1) {
+                                const room = await this.read
+                                    .getMessageReader()
+                                    .getRoom(messageId);
+                                if (room) {
+                                    const newMessage = this.modify
+                                        .getCreator()
+                                        .startMessage()
+                                        .setSender(AppSender)
+                                        .setRoom(room)
+                                        .setText(
+                                            `"${newBoardname}" Whiteboard name is already taken. Try different name`
```

Comment:
check if bold boardname works and change message to ""${newBoardname}" Whiteboard name is already taken. Please try using a different name."

Reply:
ok

Reply:
![Screenshot from 2023-12-22 23-37-07](https://github.com/RocketChat/Apps.Whiteboard/assets/108853577/05254298-b8b7-45db-abf1-f6024117276f)

is this looking good, i have made the boardname bold